operator/validator/controller.go

```diff
@@ -1300,7 +1304,10 @@ func SetupCommitteeRunners(
 			options.OperatorSigner,
 			valCheck,
 		)
-		return crunner.(*runner.CommitteeRunner)
+		if err != nil {
+			return nil, err
+		}
+		return crunner.(*runner.CommitteeRunner), nil
```

Comment:
```suggestion
		return crunner.(*runner.CommitteeRunner), err
```

Reply:
Thanks. Fixed

Reply:
@y0sher @AKorpusenko won't this panic on error though?
its safer to assume that funcs return nil value when there's an error and avoid type assertion in that case (because type assertion on nil is a panic IIRC)

Reply:
Returned back as it was.